source/extensions/common/wasm/wasm.cc

```diff
@@ -495,13 +502,57 @@ static void createWasmInternal(const VmConfig& vm_config, PluginSharedPtr plugin
                  .value_or(code.empty() ? EMPTY_STRING : INLINE_STRING);
   }
 
-  auto callback = [wasm, plugin, cb, source, allow_precompiled = vm_config.allow_precompiled(),
+  auto callback = [vm_config, scope, &cluster_manager, &dispatcher, plugin, cb, source,
+                   allow_precompiled = vm_config.allow_precompiled(),
```

Comment:
`vm_config` is now passed in, so you probably don't need to pass `allow_precompiled` separately.

Reply:
Done